packages/Heading/Heading.modules.scss

```diff
@@ -10,63 +11,58 @@
   composes: wordBreak from '../../shared/styles/Typography.modules.scss';
   composes: noSpacing from '../../shared/styles/Spacing.modules.scss';
 
-  @include helvetica-neue-light-45;
+  @include mq($until: md) {
```

Comment:
Perhaps this is personal preference, but I'd rather have a single media query instead of two; like this:

```css
.h1 {
  composes: ...;

  @include helvetica-neue-light-45;
  
  font-size: 1.75rem;
  line-height: 1.29;
  letter-spacing: -1.6px;

  .sup {
    font-size: 1.25rem;
    top: -1em;
  }
  
  @include mq($from: md) {
    font-size: 2.75rem;
    ...
  }
}
```